packages/workflow/webpack.config.js

```diff
@@ -210,8 +215,9 @@ const plugin = (settings) => {
   }
 
   // TODO: set up persistent cache options https://webpack.js.org/guides/build-performance/#persistent-cache
-
-  return config;
+  return _merge(configBase, overrides);
```

Comment:
I'm a fan of merging into a brand new object, that way we aren't mutating `configBase` itself. What do you think about changing this to `_merge({}, configBase, overrides);`?